cloudant-client/src/main/java/com/cloudant/client/api/ClientBuilder.java

```diff
@@ -607,4 +610,76 @@ public ClientBuilder readTimeout(long readTimeout, TimeUnit readTimeoutUnit) {
         return this;
     }
 
+    /**
+     * Sets Cloudant client credentials by inspecting the VCAP_SERVICES Cloud Foundry environment
+     * variable. The VCAP_SERVICES variable contains connection information to access a Bluemix
+     * service instance.
+     * <P>
+     * Note: Only a single Cloudant service must exist in VCAP_SERVICES.
```

Comment:
I think we should call out that an exception is going to be thrown if there is more than one cloudant service in the VCAP. So We'd say something like:

> @throws IllegalArgumentException If there is more than one cloudant service present in the VCAP_SERVICES.